core/java8actionloop/Dockerfile

```diff
@@ -31,13 +31,13 @@ RUN curl -sL \
   https://github.com/apache/openwhisk-runtime-go/archive/{$GO_PROXY_RELEASE_VERSION}.tar.gz\
   | tar xzf -\
   && cd openwhisk-runtime-go-*/main\
-  && GO111MODULE=on go build -o /bin/proxy
+  && GO111MODULE=on CGO_ENABLED=0 go build -o /bin/proxy
 
 # Use AdoptOpenJDK's JDK8, OpenJ9, ubuntu
 FROM ibm-semeru-runtimes:open-8u332-b09-jdk-focal
 
 # select the builder to use
-ARG GO_PROXY_BUILD_FROM=release
+ARG GO_PROXY_BUILD_FROM=source
```

Comment:
@style95 seems change GO_PROXY_BUILD_FROM to `source` is not better, but in order to support array result, must  use go runtime's upstream master code.

Can we release a new version for go runtime? e.g. https://github.com/apache/openwhisk-runtime-java/blob/master/core/java8actionloop/Dockerfile#L29
Change

Reply:
We could do a wave of runtime releases, starting the the go runtime.   Would be reasonable to plan to get the go runtime released this week (takes 72 hours for the vote).

Reply:
@dgrove-oss , due to go runtime : 1.20.0 is released, i have updated all runtime prs(support array result)'s Dockerfile to apply this.